lib/castle_web/search.ex

```diff
@@ -0,0 +1,47 @@
+defmodule CastleWeb.Search do
+
+  import Ecto.Query
+
+  def prefix_search(query) do
+
+    ends_with_whitespace = Regex.match?(~r/\s$/, query)
+    query = String.trim(query)
+
+    query = query
```

Comment:
I've been pretty inconsistent with my pipe formatting, so i'm hardly one to complain ...

But the elixir styleguide suggests a [slightly different](https://github.com/christopheradams/elixir_style_guide#pipe-operator) way to indent those.

Reply:
And now that i'm looking at formatting, there's been a `mix format` command that implements a lot of those styleguide rules.

And I pledge to do better to follow that styleguide going forward!

Reply:
Good point! I like the style that the formatter produces.